Sources/IBLinterFrontend/Commands/ValidateCommand.swift

```diff
@@ -71,6 +78,8 @@ struct ValidateOptions: OptionsProtocol {
             <*> mode <| Option(key: "reporter", defaultValue: nil, usage: "the reporter used to log errors and warnings")
             <*> mode <| Option(key: "script", defaultValue: nil, usage: "custom IBLinterfile.swift")
             <*> mode <| Option(key: "config", defaultValue: nil, usage: "the path to IBLint's configuration file")
+            // swiftlint:disable:next line_length
```

Comment:
if it work prefer use swiftlint:disable:this line_length by putting it at the end of the line